corehq/apps/data_dictionary/models.py

```diff
@@ -71,6 +82,15 @@ class DataType(models.TextChoices):
     )
     group = models.TextField(default='', blank=True)
     index = models.IntegerField(default=0, blank=True)
+    group_obj = models.ForeignKey(
+        CasePropertyGroup,
+        on_delete=models.CASCADE,
+        related_name='properties',
+        related_query_name='property',
+        db_column="group_id",
```

Comment:
Or does this make it easier to eventually migrate the `ForeignKey` to be just called `group`, since that'd result in a `group_id` column in the DB to store the related model's primary key?

Reply:
This makes it easier to rename the `group_obj` field to `group`.